source/common.c

```diff
@@ -61,6 +62,58 @@ void *aws_mem_acquire(struct aws_allocator *allocator, size_t size) {
     return mem;
 }
 
+#define AWS_ALIGN_ROUND_UP(value, alignment) (((value) + ((alignment)-1)) & -(alignment))
```

Comment:
[trivial] prefer `~alignment`rather than `-alignment` so you're not relying on whether the machine uses 1 or 2 complement.